benchmarks/client-nav/solid/client-nav.bench.tsx

```
@@ -0,0 +1,154 @@
+      }
+
+      return (
+        <div>
+          <div style={{ display: 'none' }}>{rootScore()}</div>
+          <div style={{ display: 'none' }}>
+            {Array.from({ length: LINK_COUNT }, (_, index) => (
+              <Link
+                to="/$id"
+                params={{ id: String(index) }}
+                search={{ n: index }}
+              >
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)